mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/dirtiness/building/impl/PojoIndexingDependencyCollectorValueNode.java

```diff
@@ -56,27 +56,36 @@
 
 	private static final Log log = LoggerFactory.make( Log.class, MethodHandles.lookup() );
 
-	private final BoundPojoModelPathValueNode<?, P, V> modelPathFromRootEntityNode;
+	private final PojoIndexingDependencyCollectorPropertyNode<?, P> parentNode;
+	private final BoundPojoModelPathValueNode<?, P, V> modelPathFromLastTypeNode;
```

Comment:
What's the difference between `modelPathFromLastEntityNode` and `modelPathFromLastTypeNode`?

I think some javadoc would help as it's pretty hard to follow. Same in the class above.

Reply:
Perfect, it's a lot clearer now.